data/config_options.yml

```
@@ -498,8 +498,8 @@ options:
       - `file` (default)
         - Write all AppSignal logs to the file system.
       - `stdout` (default on [Heroku](http://heroku.com/))
-        - Print AppSignal logs in the parent process' STDOUT instead of to a file.
-          Useful with hosting solutions such as container systems and Heroku.
+        - Print AppSignal logs in the parent process' STDOUT.
+          Useful with hosting solutions such as container systems and Heroku. The log file will still be created and agent will still write to it even if you set the log setting to STDOUT.
 
       -> **Note**: At this time only the AppSignal integration supports this feature and the [AppSignal agent](/appsignal/how-appsignal-operates.html), which is used by the integration, does not yet support this.
```

Just reading this change again and I notice this has already been described on this line here.

```suggestion
      -> **Note**: At this time only the AppSignal integration supports the "stdout" log feature and the [AppSignal agent](/appsignal/how-appsignal-operates.html), which is used by the integration, does not yet support this.
```

---

When I read that notice, it was not clear that's what it meant (agent still writes to the log file), maybe we should then change the wordings of that notice, wdyt?

---

Yes it needs to be more clear. I left another suggestion below, but feel free to improve upon it.

```suggestion
      -> **Note**: At this time only the AppSignal integration supports the "stdout" log output feature. The [AppSignal agent](/appsignal/how-appsignal-operates.html), which is used by the integration, does not support "stdout" log output. It will always write to the "appsignal.log" file.
```